packages/strapi/lib/services/metrics/truthy-var.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const isTruthyEnvVar = val => {
+  if (val === null || val === undefined) return false;
+
+  if (val === true) return true;
+
+  if (val.toString().toLowerCase() === 'true') return true;
+  if (val.toString().toLowerCase() === 'false') return false;
+
+  if (val === 1) return true;
+
+  return false;
+};
+
+module.exports = isTruthyEnvVar;
```

Comment:
The code is more consice but is less understandable in my opinion. You will need to understand it will cast boolean types to string 'true' so it matches 'true'.

Reply:
But the environment variables are always string no? So there would be no cast actually

Reply:
The function is just taking a param as input. We cannot assume it will be a string.I will rename the function for clarity.